daemon/daemon.go

```diff
@@ -442,7 +442,17 @@ configRetry:
 		log.Infof("Starting the Typha connection")
 		err := typhaConnection.Start(context.Background())
 		if err != nil {
-			log.WithError(err).Fatal("Failed to connect to Typha")
+			retry := 0
+			for err != nil && retry < 10 {
+				// Set Ready and Live to false
+				healthAggregator.Report(healthName, &health.HealthReport{Live: false, Ready: false})
+				err = typhaConnection.Start(context.Background())
+				log.WithError(err).Warn("Retrying to start Typha")
```

Comment:
Please drop this log to debug or we'll emit just as many high-level logs as we used to.

Reply:
Ack